youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -542,6 +545,29 @@ def can_merge(self):
         return True
 
 
+class FFmpegConcatPP(FFmpegPostProcessor):
+    def run(self, info):
+        filename = info['filepath']
+
+        list_filename = prepend_extension(filename, 'list')
```

Comment:
The filename will end up as `.list.mkv` which doesn't make sense. It should be either `.mkv.list` or just `.list`